clients/acm.d.ts

```diff
@@ -0,0 +1,350 @@
+import {Request} from '../lib/request';
+import {Response} from '../lib/response';
+import {AWSError} from '../lib/error';
+import {Service} from '../lib/service';
+declare class ACM extends Service {
+  /**
```

Comment:
Another reason to have a constructor definition for each service class is to allow code completion for API version dates (by defining the optional `apiVersion` as an enum in `ServiceParams`), especially if we want to encourage users to specify API versions when constructing the service clients.